.sync.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+mock_with: ':rspec'
+

```

Comment:
can you add the other items like here:
https://github.com/MiamiOH/puppet-duo_authproxy/blob/master/.sync.yml

Reply:
specifically the gitlab-ci and travis stuff